tests/fs/test_nodefs_list_dir.c

```diff
@@ -0,0 +1,28 @@
+#include <dirent.h>
+#include <emscripten.h>
+#include <stdio.h>
+
+
+void list_dir(const char *path) {
+    struct dirent *entry;
+    DIR *dir = opendir(path);
+    if (dir == NULL) {
+        return;
+    }
+    while ((entry = readdir(dir)) != NULL) {
+        printf("%s\n",entry->d_name);
+    }
+    closedir(dir);
+}
+
+
+int main() {
```

Comment:
Maybe a comment here saying what we are testing?  Something like "Test that readdir works directly on a mount point"?